torrent.go

```diff
@@ -710,6 +710,32 @@ func (t *Torrent) bitfield() (bf []bool) {
 	return
 }
 
+func (t *Torrent) CompletedPiecesBitfield() []bool {
+	return t.bitfield()
+}
+
+// Returns a list containing the number of peers that have a specific piece available
+func (t *Torrent) PieceAvailabilityList() (connCount int, connBitfieldSum []int) {
```

Comment:
Perhaps we should just expose the Torrent connections list, and let people do this kind of thing on their own?

Reply:
I think that would be a good idea as currently there is little to no insight into specific peer status from an external library